vpr/src/place/initial_placement.h

```diff
@@ -3,6 +3,24 @@
 
 #include "vpr_types.h"
 
+/**
+ * @brief Used to assign each block a score for how difficult it is to place. 
+ * The higher numbers indicate a block is expected to be more difficult to place.
+ * Hence, initial placement tries to place blocks with higher scores earlier.
+ */
+struct t_block_score {
+    int macro_size = 0; //How many members does the macro have, if the block is part of one - this value is zero if the block is not in a macro
+
+    //The number of tiles NOT covered by the block's floorplan constraints.
```

Comment:
Is this normalized or not?

Reply:
Should normalize if it isn't right now (could divide by number of tiles in device, could also multiple after that by some constant in your comparison lambda).